internal/cli/down.go

```diff
@@ -52,19 +51,22 @@ func (c downCmd) run(ctx context.Context, args []string) error {
 		logger.Get(ctx).Infof("error deleting k8s entities: %v", err)
 	}
 
-	var dcManifests []model.Manifest
+	var dcConfigPath string
 	for _, m := range manifests {
-		if m.IsDockerCompose() {
-			dcManifests = append(dcManifests, m)
+		if dcInfo, ok := m.DCInfo(); ok {
```

Comment:
a common Go idiom is to have a Empty() or IsZero() method for the "zero" value of a struct...would that make more sense here?

Reply:
OH HEY WAIT @nicks @jazzdan i remember why i wanted to do it this way -- b/c if if you get back an empty dcInfo from `m.DCInfo` there's no way of telling whether
a. there's nothing in `m.deployInfo` (i.e. it's safe to overwrite with docker stuff), or
b. there's something ELSE in `m.deployInfo` (like k8s stuff) that failed the type check

I imagine this `dcInfo, ok` syntax mirroring that of a type cast. I can comment with this explanation, but -- does that make more sense / sound more acceptable?

Reply:
Ok